[No QA] Update help docs for Release 1: Automatic report reassignment on workflow changes (#97799)
Expensify/App: docs/articles/new-expensify/workspaces/Managing-Workspace-Members.md

**Note:** Workspace admins and People admins can remove members. People admins can only remove **Members** and **Auditors** — they cannot remove admins or other scoped admin roles.

> **Note:** If you remove a member who is an approver on reports that are still pending approval, those reports are automatically reassigned to the workspace owner. Learn more about [automatic approver reassignment](/articles/new-expensify/workspaces/Add-Approvals).

Reviewer: ```suggestion
**Note:** If you remove a member who is an approver on an Outstanding report, the workspace owner replaces them as the approver. Learn more about [automatic approver reassignment](/articles/new-expensify/workspaces/Add-Approvals).
```
